### DISCUSSION TITLE
Is it possible to get the size of the ambient cache?

### DISCUSSION THREAD

**Discussion by ludvigeriksson** [rnmapbox/maps#1863] [Q&A]
I can't seem to find a method for this in the documentation.

**Comment by VolkerLieber** [comment]
The ambient cache size is not exported by mapbox atm. You can find the current implementation here: 
**.CPP**
[https://github.com/mapbox/mapbox-gl-native/blob/b8edc2399b9640498ccbbbb5b8f058c63d070933/platform/default/src/mbgl/storage/offline_database.cpp](https://github.com/mapbox/mapbox-gl-native/blob/b8edc2399b9640498ccbbbb5b8f058c63d070933/platform/default/src/mbgl/storage/offline_database.cpp)
**.H**
[https://github.com/mapbox/mapbox-gl-native/blob/b8edc2399b9640498ccbbbb5b8f058c63d070933/platform/default/include/mbgl/storage/offline_database.hpp](https://github.com/mapbox/mapbox-gl-native/blob/b8edc2399b9640498ccbbbb5b8f058c63d070933/platform/default/include/mbgl/storage/offline_database.hpp)
